### PR TITLE
SME with IIIF manifest

### PR DESCRIPTION
**Pull request by Dananji**
Update Gemfile after `iiif_manifest` gem is released with the changes to manifest generation with `rendering` property at canvas level.

### CODE REVIEW

File: Gemfile
Comment (cjcolvar) [inline Gemfile:52]: `iiif_manifest` 1.3.0 was released last week which should include your fixes.
```suggestion
gem 'iiif_manifest', '~> 1.3'
```

**Comment by Dananji** [comment]
Do not merge this yet, streaming in SME is not working for me locally.

**Review by cjcolvar — APPROVED** [review]
I think this looks good, but we should wait until 7.5.1 is released before merging.

**Review by cjcolvar — APPROVED** [review]

**Pull request merged**
